run-tests.php

```diff
@@ -1940,26 +1930,23 @@ function run_test($php, $file, $env)
 			}
 		}
 
-		$org_headers = $headers;
-		$headers = array();
 		$output_headers = array();
 
 		foreach($want as $k => $v) {
 
-			if (isset($org_headers[$k])) {
-				$headers = $org_headers[$k];
-				$output_headers[] = $k . ': ' . $org_headers[$k];
+			if (isset($headers[$k])) {
+				$output_headers[] = $k . ': ' . $headers[$k];
 			}
 
-			if (!isset($org_headers[$k]) || $org_headers[$k] != $v) {
+			if (!isset($headers[$k]) || $headers[$k] != $v) {
 				$failed_headers = true;
```

Comment:
Refactoring away $org_headers appears to be fine, but shouldn't also the [initialization of $headers](https://github.com/php/php-src/blob/php-7.0.8/run-tests.php#L1955) be changed to array()? That looks wrong, anyway.